### DISCUSSION TITLE
loading pretrained about the class MMDetBackbone

### DISCUSSION THREAD

**Discussion by Shuntw6096** [facebookresearch/detectron2#3832] [Q&A] [answered]
I find the class MMDetBackbone has a keyword argument `pretrained_backbone`, but it seems that function about loading `pretrained_backbone` is not implemented, will it fix?
https://github.com/facebookresearch/detectron2/blob/e9cbe90212102d9a2538559b673b66cb98644a6a/detectron2/modeling/mmdet_wrapper.py#L59-L60
because I use `SwinTransformer` from `mmdet`, and loading pretrained was implemented in `mmdet`

https://github.com/open-mmlab/mmdetection/blob/5e246d5e3bc3310b5c625fb57bc03d2338ca39bc/configs/swin/mask_rcnn_swin-t-p4-w7_fpn_1x_coco.py#L26

**Comment by ppwwyyxx** [comment, marked as answer]
You're right. The documentation is for older mmdet and should be removed now. The correct way to initialize is to use `init_cfg`.
